github/gists.go

```diff
@@ -52,6 +52,30 @@ func (g GistFile) String() string {
 	return Stringify(g)
 }
 
+// GistCommit represents a commit on a gist.
+type GistCommit struct {
```

Comment:
Fair enough. I think those are issues that should also be resolved.

It's possible the URL field is relatively new and didn't exist at the time those structs were added. Or maybe it was missed during the original code review.

I am not aware of any special rules or reasons to exclude the URL field.